packages/xrpl/HISTORY.md

```diff
@@ -3,6 +3,9 @@
 Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xrpl-announce) for release announcements. We recommend that xrpl.js (ripple-lib) users stay up-to-date with the latest stable release.
 
 ## Unreleased Changes
+* Fix for parseTransactionFlags to no longer modify a passed in transaction when returning a parsed mapping
```

Comment:
This should be under a heading that matches the actions - see https://keepachangelog.com/en/1.1.0/